src/Syntax.hs

```diff
@@ -265,9 +289,12 @@ makePrisms ''Lit
 makeLenses ''Skolem
 makeLenses ''TyBinder
 
+instance Spanned (Ann p) => Spanned (Binding p) where
+  annotation (Binding _ _ _ x) = annotation x
```

Comment:
Could possibly be replaced with accessor.

Reply:
Fixed!